pkg/apis/options/legacy_options.go

```diff
@@ -472,7 +472,7 @@ type LegacyProvider struct {
 
 	KeycloakGroups           []string `flag:"keycloak-group" cfg:"keycloak_groups"`
 	AzureTenant              string   `flag:"azure-tenant" cfg:"azure_tenant"`
-	BitbucketTeam            string   `flag:"bitbucket-team" cfg:"bitbucket_team"`
+	BitbucketWorkspace       string   `flag:"bitbucket-workspace" cfg:"bitbucket_workspace"`
```

Comment:
This would be a breaking change, is there a way to avoid this? Perhaps we add a new option and deprecate the old one, but copy the old value to the new one if specified?

Reply:
Yes, that should do. Will update

Reply:
Hopefully that isn't too challenging given we already have the legacy to new config conversion in place